README.md

```diff
@@ -1 +1,194 @@
-# backend_capstone
\ No newline at end of file
+<div align="center">
```

Comment:
- [ ] Kindly, include a "tests" section on your project stating how to run your tests properly. 👍